lib/msal-angular/src/msal.guard.ts

```diff
@@ -175,6 +193,12 @@ export class MsalGuard implements CanActivate, CanActivateChild, CanLoad {
             );
     }
 
+    includesCode(path: string): boolean {
+        return path.indexOf("/code") > -1 ||
```

Comment:
Does this need the =?
```suggestion
        return path.indexOf("/code=") > -1 ||
```

Reply:
With hash routing, the parsed path does not include the `=` (see the comment in the guard above).

Reply:
How does that work? How are the keys and values separated?

Reply:
I'm a bit confused about this as well.

Reply:
> How does that work? How are the keys and values separated?

Sorry, referring to how the Angular router parses the url, not MSAL. 

When the Angular router (with hash routing) sees `/#code=123`, it parses it to `/code`

Reply:
I guess this is assuming the app has a `/code` route registered. Lemme see what happens if a `/code` path isn't registered.

Reply:
Would that mean that someone who has a `codes` route or anything else that starts with `code` would have to use path routing to be compatible? If so, that should be documented.

Reply:
Hmm good point, will make this smarter such that `/codes` and `/code` are treated different.

Reply:
When using hash routing, `/code` must be registered, and updated logic to ensure route must end with `/code`.